File: division.py
x=int(input("valeur 1 ? ")) # valeur 1 saisie convertie en entier.
y=int(input("valeur 2 ? ")) # valeur 2 saisie convertie en entier.

if(y==0):
  print("pas de resultat, division par zero")
else:
    resultat = x/y

print ("la division de ",x,"/",y,"est",resultat)

File: division.c
#include <stdio.h>
#include <stdlib.h>
int main(){
		//definir
		float a,b,resultat;
		//fin definir

		//debut
		printf("donner la valeur de a svp : \n");
		scanf ("%f",&a);
		printf("donner la valeur de b svp : \n");
		scanf ("%f",&b);
		if(b==0){
			printf("pas de resultat, division par zero\n");
			return 1;
		}else{
			resultat = a/b;
		}
		printf("le resultat de %f/%f est : %f\n",a,b,resultat);
		//fin
		return 1;
}